tests/php/Shortcodes/ImageShortcodeProviderTest.php

```diff
@@ -380,4 +482,39 @@ public function testCreateImageTag(string $expected, array $attributes)
     {
         $this->assertEquals($expected, ImageShortcodeProvider::createImageTag($attributes));
     }
+
+    /**
+     * This method will assert that the $tag will contain an image with specific attributes and values
+     *
+     * @param array $attrs Key pair values of attributes and values.
+     *                     If the value is FALSE we assume that it is not present
+     */
+    private function assertImageTag(array $attrs, string $shortcode, $message = ""): void
```

Comment:
Instead of testing whether the HTML generated by a specific shortcode exactly matches the expected HTML, this method will parse the HTML and check the individual attributes.

This is better because we shouldn't care about the order of the attributes. What we care about is whether the attribute is there or not and whether it has the correct value.